schemas/string_interpolation/transforms/email_link.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$id": "https://eq.ons.gov.uk/string_interpolation/transforms/email_link.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "email_link": {
+    "type": "object",
+    "properties": {
+      "transform": {
+        "type": "string",
+        "enum": ["email_link"]
+      },
+      "arguments": {
+        "type": "object",
+        "properties": {
+          "email_address": {
+            "pattern": "^[A-Za-z0-9._+-]+@[A-Za-z0-9.-]+.[A-Za-z.]+$"
```

Comment:
There should also be a `type` property set to `string`.
https://json-schema.org/understanding-json-schema/reference/string.html#regular-expressions